ui/src/store/RelayClient.ts

```diff
@@ -341,4 +341,46 @@ export class RelayClient {
       payload: originalContactHash,
     });
   }
+
+  /**
+   * Delete a message
+   *
+   * Delete a message by its hash, and get the status of the deletion.
+   * Message is not actually deleted, but a delete action is created.
+   *
+   */
+
+  public async deleteMessage(cellId: CellId, messageHash: ActionHash): Promise<ActionHash> {
+    return this.client.callZome({
+      cell_id: cellId,
+      zome_name: ZOME_NAME,
+      fn_name: "delete_message",
+      payload: messageHash,
```

Comment:
```suggestion
      payload,
```

nitpick but more legible